src/appleseed.studio/mainwindow/rendering/rendertab.cpp

```diff
@@ -299,6 +306,35 @@ void RenderTab::create_toolbar()
     m_picking_mode_combo->setObjectName("picking_mode_combo");
     m_toolbar->addWidget(m_picking_mode_combo);
 
+    // Create the label preceding the display combobox.
+    QLabel* display_label = new QLabel("Display:");
+    display_label->setObjectName("display_label");
+    m_toolbar->addWidget(display_label);
+
+    // Create the display combobox.
+    m_display_transform_combo = new QComboBox();
+    m_display_transform_combo->setObjectName("display_combo");
+    {
+        const char* display_name = m_ocio_config->getDefaultDisplay();
+        std::string default_transform = m_ocio_config->getDefaultView(display_name);
+
+        int default_index = 0;
+        for (int i = 0, e = m_ocio_config->getNumViews(display_name); i < e; ++i)
+        {
+            const char* name = m_ocio_config->getView(display_name, i);
+            m_display_transform_combo->addItem(name);
+
+            if (default_transform == name)
+                default_index = i;
```

Comment:
No, because we are building the menu and getting the default_index at the same time.
If we break we don't add the items after default_index.